### ISSUE TITLE
Asura damage should not be affected by Provoke, Curse and INCATKRATE

### ISSUE THREAD

**Issue by Playtester** [rathena/rathena#1823]
Provoke, Curse and INCATKRATE (e.g. from Gospel or Tarot Card) do not change the Asura damage at all.
BATKFOOD and ATKPOTION however do increase Asura damage.

This would indicate that Provoke, Curse and INCATKRATE should not actually be part of the base attack calculation but instead add / substract damage in the damage calculation.

This applies to both pre-re and re.

**Comment by anacondaqq** [comment]
xD, another issue which will be used by players & server owners for: 
"PLEASE RETURN BACK OLD BEHAVIOR !!!!11111" =) Anyway, thank you very much for future bugfix.

**Comment by Locien** [comment]
anacondaqq, there's no need to point that out. We know that a lot of people have been playing on servers with incorrect behavior which they're used to. And if you think it's a problem for your server you can always revert the change and no harm will be done.

**Comment by anacondaqq** [comment]
@Locien it's not my problem bro, it's problem of rAthena and Ragnarok Online community =) Which in final will totally destroy and RO and rA. I do not worry about it, i just warn that will be very many issues on all servers in next 4-8 weeks or less or more, and my comment not about "how bad rAthena", or "what are you doing, please stop".

My comment about very big problem for RO scena at all, not because of changes, but because of rudimentary "things" which very late to add to pre-renewal mechanic as static not changable with config options things (but not for renewal one, because renewal is very actual for now),   and as i see all of these changes in last time added not because rAthena needs it, but because 1-2 servers need it =) This is mega funny thing and dramatic at the same time, which will in result split emulator for pre-re and renewal part (remember my words) when new "power" will born. 

And be sure, new power with new emulator will born, because i monitor very large amount of servers (pre-re based) in RO scena, and around in my own count and opinion 80% of them has the same issues, the same bugreports, the same "kind words" to staff. And I can see significant pressure right now in RO scena because someone "building" only fixes for specialized servers, and do not care about other one or totally ignoring them by saying "if you need - revert changes". 

Good luck guys.

**Comment by Playtester** [comment]
Nobody does changes for a server, we do changes because it's official behavior.

If anyone is to blame, it's the 50000 players that didn't report the issue much earlier.

**Comment by Playtester** [comment]
These status changes also don't change Spiral Pierce damage (when used by mobs it uses base attack). For example Strength Tarot Card would half Seyren's normal damage but Spiral Pierce damage is still the same.

**Comment by Playtester** [comment]
It gets really complicated. Shield Boomerang is not affected by the bonus, but Smite (Shield Chain) is...

**Comment by keitenai** [comment]
oh boy... this is going to take a lot of explanation to players and these players as we all know will be split into 8:10 to say "we don't like it / we quit / you're noob" and many more that will hurt dev's feelings... T_T

**Comment by Playtester** [comment]
Might have been fixed in 4247bc5.

I think Rapid Smiting (Shield Chain) being affected by is was a false report. I only recently realized that there is damage variance if your shield is heavy. Also, Curse actually affects its damage, but not because of the -25% ATK penalty but because it sets LUK to 0.

I'm running some tests to check if Spiral Pierce was fixed through this or we need to put it in the list of skills unaffected by ATKpercent.

Lord Knight 90/69 Javelin+5, Spear Mastery 10 vs. Golem

Spiral Pierce 1: 73x5
Cursed + Spiral Pierce 1: 73x5
Dynamo + Spiral Pierce 1: 73x5
